routes/routes.js

```diff
@@ -1,4 +1,6 @@
+const users = require("./users.route");
 module.exports = function(app){
```

Comment:
Where are auth routes

Reply:
This is only registration. Not authentication